src/main/java/frc/utils/math/FieldMath.java

```diff
@@ -0,0 +1,76 @@
+package frc.utils.math;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+import frc.constants.MathConstants;
+import frc.constants.field.Field;
+
+public class FieldMath {
+
+	public static Translation2d getRelativeTranslation(Translation2d relativeTo, Translation2d reference) {
+		return reference.minus(relativeTo);
+	}
+
+	public static Translation2d getRelativeTranslation(Pose2d relativeTo, Translation2d reference) {
+		return getRelativeTranslation(relativeTo.getTranslation(), reference).rotateBy(relativeTo.getRotation().unaryMinus());
+	}
+
+	public static Rotation2d getMirroredAngle(Rotation2d angle) {
```

Comment:
```suggestion
	public static Rotation2d mirrorAngle(Rotation2d angle) {
```
not a must i prefer it like this